tests/processes/ProcessingTest.py

```diff
@@ -572,7 +572,7 @@ def test_failing_action(self):
         class FailingTestAction(ResultAction):
 
             def apply(self, *args, **kwargs):
-                raise RuntimeError("YEAH THAT'S A FAILING BEAR")
+                raise RuntimeError('YEAH THAT\'S A FAILING BEAR')
```

Comment:
This needs double quotes barbecue of the `that's`. (I think double quotes are preferred to escaping, not sure)